src/index.js

```diff
@@ -191,7 +204,10 @@ export default class RolloverTodosPlugin extends Plugin {
           file: file,
           oldContent: `${dailyNoteContent}`
         }
-        const todos_todayString = `\n${todos_today.join('\n')}`
+        todos_todayString = `\n`
```

Comment:
I am surprise you can change `const todos_todayString = ...` with `todos_todayString = ...`. Don't you need a `let todos_todayString = ...` here?